nevergrad/functions/images/imagelosses.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from koncept.models import Koncept512
+
+
+class _ImageLoss:
```

Comment:
```suggestion
class ImageLoss:
```
(and propagate)

this will be used for typing